capture/capture.go

```diff
@@ -190,18 +171,18 @@ func (l *Listener) Filter(ifi NetInterface) (filter string) {
 	if l.trackResponse {
 		dir = " "
 	}
-	filter = fmt.Sprintf("(%s%s%s)", l.Transport, dir, port)
```

Comment:
The reason why transport was there is that I wanted to keep a chance to support another transport in future, like udp.

Reply:
Oh! I will revert it